api_tests/harness.ts

```diff
@@ -19,6 +19,8 @@ const rlp = readline.createInterface({
     terminal: true,
 });
 
+const IS_CI = process.env.CI ?? Boolean(JSON.parse(process.env.CI));
```

Comment:
It would be nice to be able to trigger this output locally also.

Reply:
You can, use `--verbose`.